src/components/navbars/DesktopNav.jsx

```diff
@@ -1,32 +1,39 @@
+import ProfileNav from './ProfileNav';
 import loginIcon from '../../assets/icons/login.png';
 import searchIcon from '../../assets/icons/search.png';
+import { useAuth } from '../../AuthProvider';
 
 const DesktopNav = ({ onSearch, onLogin, onJoin, stateSearch, stateLogin, stateJoin }) => {
```

Comment:
The `state` parameter names are too abstract. What is state? If we are expecting a boolean let's follow conventional naming.

Reply:
I fixed to isSearch, isLogin, isJoin